src/applications/disability-benefits/all-claims/analytics-events.js

```diff
@@ -0,0 +1,32 @@
+import {
+  ITF_FETCH_SUCCEEDED,
+  ITF_FETCH_FAILED,
+  ITF_CREATION_SUCCEEDED,
+  ITF_CREATION_FAILED,
+} from './actions';
+
+const analyticsEvents = [
+  {
+    action: ITF_CREATION_SUCCEEDED,
+    event: 'disability-526EZ-itf-created',
+  },
+  {
+    action: ITF_CREATION_FAILED,
+    event: 'disability-526EZ-itf-not-creation',
+  },
+  {
+    action: ITF_FETCH_SUCCEEDED,
+    event: 'disability-526EZ-itf-retrieved',
+  },
+  {
+    action: ITF_FETCH_FAILED,
+    event: store => {
```

Comment:
Not sure where to put this, but what do you think about documenting the signature of an `event` function somewhere we're likely to see if next time we go to add events?

Reply:
I'm all for documenting, but I'd like to make sure it's discoverable. Any suggestions?